ngraph/test/onnx/onnx_import_with_editor.in.cpp

```diff
@@ -0,0 +1,101 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+// clang-format off
+#ifdef ${BACKEND_NAME}_FLOAT_TOLERANCE_BITS
+#define DEFAULT_FLOAT_TOLERANCE_BITS ${BACKEND_NAME}_FLOAT_TOLERANCE_BITS
+#endif
+#ifdef ${BACKEND_NAME}_DOUBLE_TOLERANCE_BITS
+#define DEFAULT_DOUBLE_TOLERANCE_BITS ${BACKEND_NAME}_DOUBLE_TOLERANCE_BITS
+#endif
+// clang-format on
+
+#include "gtest/gtest.h"
+#include "onnx_editor/editor.hpp"
+#include "ngraph/ngraph.hpp"
+#include "util/test_case.hpp"
+#include "util/test_control.hpp"
+#include "util/engine/test_engines.hpp"
+
+using namespace ngraph;
+
+static std::string s_manifest = "${MANIFEST}";
+
+using Inputs = std::vector<std::vector<float>>;
```

Comment:
I believe this is a copy-and-paste artifact but we don't use those aliases any more. Not in this file at least.

Reply:
`s_manifest` is needed for test skipping from manifest file AFAIK

Reply:
I was referring to Inputs and Outputs aliases, no the string above.